src/main/java/com/brainpix/post/controller/IdeaMarketController.java

```diff
@@ -0,0 +1,61 @@
+package com.brainpix.post.controller;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.brainpix.api.ApiResponse;
+import com.brainpix.post.dto.GetIdeaCommentListDto;
+import com.brainpix.post.dto.GetIdeaDetailDto;
+import com.brainpix.post.dto.GetIdeaListDto;
+import com.brainpix.post.dto.GetPopularIdeaListDto;
+import com.brainpix.post.service.IdeaMarketService;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+@RequestMapping("/api/ideas")
```

Comment:
여기서 /api만 없애주시면 될거같습니다!
```suggestion
@RequestMapping("/ideas")
```